gcp_variant_transforms/vcf_to_bq.py

```diff
@@ -186,15 +186,18 @@ def _shard_variants(known_args, pipeline_args, pipeline_mode):
   with beam.Pipeline(options=options) as p:
     variants = _read_variants(
         known_args.all_patterns, p, known_args, pipeline_mode)
-    call_sample_ids = (variants
-                       | 'CombineCallSampleIds' >>
-                       combine_call_sample_ids.CallSampleIdsCombiner())
+    sample_ids = (variants
+                  | 'CombineSampleIds' >>
+                  combine_sample_ids.SampleIdsCombiner())
+    # TODO(tneymanov): Annotation pipeline currently stores sample IDs instead
+    # of sample names in the the sharded VCF files, which would lead to double
+    # hashing of samples. Needs to be fixed ASAP.
```

Comment:
Again please submit a Github issue and refer to this PR and this TODO. Feel free to use one issue for both this and bq_to_vcf TODO.